public/data/caotanduc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Cao Tan Duc",
+  "type": "personal",
+  "bio": "Student",
+  "avatar": "https://github.com/caotanduc.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/caotanduc",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://facebook.com/tan.duc.92317/",
+      "icon": "Facebook"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "University",
+      "date": "September 2020",
+      "icon": "book",
+      "color": "grey",
+      "description": "Start studying at Ho Chi Minh University of Science - Vietnam"
```

Comment:
Tense error 😅 

```suggestion
      "description": "Started studying at Ho Chi Minh University of Science - Vietnam"
```

Reply:
👍 Thanks for your help. I am not good at English so I used Google translate for this sentence, but it doesn't work well with my Mother tongue.